cgi/reset_password.pl

```diff
@@ -72,7 +72,7 @@
 	# Is it an email?
 
 		if ($id =~ /\@/) {
-			my $emails_ref = retrieve("$data_root/users_emails.sto");
+			my $emails_ref = retrieve("$data_root/users/users_emails.sto");
```

Comment:
To ease transition on production, I propose to already move the files to users/ and orgs/ but symlink them in the upper directory.

Reply:
pour l'info je l'ai fait en preprod, pour tester.

Reply:
sounds good